.github/workflows/windows.yml

```diff
@@ -119,6 +159,42 @@ jobs:
         name: build
     - name: Check directory tree (for debugging purposes)
       run: tree .
+    - name: Add vcpkg installed package dirs to path
+      run: |
+        echo "::add-path::${{ github.workspace }}\vcpkg\installed\x64-windows\bin"
+        echo "::add-path::${{ github.workspace }}\vcpkg\installed\x64-windows\debug\bin"
+    - name: Check installed vcpkg libs
+      run: |
+        dir "${{ github.workspace }}\vcpkg\installed\x64-windows\bin"
+        dir "${{ github.workspace }}\vcpkg\installed\x64-windows\debug\bin"
+    - name: Add build libs to path
+      run: |
+        echo "::add-path::${{ github.workspace }}\build\src\lib\ecore"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\ecore_con"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\eet"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\efl"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\eina"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\emile"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\eo"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\evil"
+        echo "::add-path::${{ github.workspace }}\build\subprojects\getopt"
+        echo "::add-path::${{ github.workspace }}\build\subprojects\pcre-8.37"
+        echo "::add-path::${{ github.workspace }}\build\src\lib\ector"
+    - name: Check built EFL libs
+      run: |
+        dir "${{ github.workspace }}\build\src\lib\ecore"
+        dir "${{ github.workspace }}\build\src\lib\ecore_con"
+        dir "${{ github.workspace }}\build\src\lib\eet"
+        dir "${{ github.workspace }}\build\src\lib\efl"
+        dir "${{ github.workspace }}\build\src\lib\eina"
+        dir "${{ github.workspace }}\build\src\lib\emile"
+        dir "${{ github.workspace }}\build\src\lib\eo"
+        dir "${{ github.workspace }}\build\src\lib\evil"
+        dir "${{ github.workspace }}\build\subprojects\getopt"
+        dir "${{ github.workspace }}\build\subprojects\pcre-8.37"
+        dir "${{ github.workspace }}\build\src\lib\ector"
+    - name: Check path
```

Comment:
So for now on, for each lib we enable we need to add its dll here?

Reply:
Could you sort those, please? =D

Reply:
Yes (actually let me do a quick test before that) and yes.

Reply:
`check` dll is don't need to be set like this too?

Reply:
oh, our dependencies don't need that it seems.

Reply:
Yup, they can be removed, to the first answer changes to "Nope" :D

Reply:
(For reference: https://github.com/expertisesolutions/efl/runs/950753122?check_suite_focus=true#step:20:1)

Reply:
hummm, but ector is still needed?

Reply:
Didn't see ector there. Should be fixed now.

Reply:
oh, right it was fixed. So why dll's were needed before and now they are not?